types/chai/chai-tests.ts

```diff
@@ -1405,7 +1405,10 @@ suite('assert', () => {
     test('assert', () => {
         const foo = 'bar' as string;
         assert(foo === 'bar', 'expected foo to equal `bar`');
+    });
 
+    test('assertFailure', () => {
+        const foo = 'bar' as string;
```

Comment:
Not sure why this test case was split. This doesn't appear to be related to the PR.

Reply:
This test case is split because of the new type information introduced by the assertion ``assert(foo === 'bar', ...);``:

```
Error: /home/felis/Projects/DefinitelyTyped/types/chai/chai-tests.ts:1408:7
ERROR: 1408:7   comment-format  comment must start with a space
ERROR: 1410:7   comment-format  comment must start with a space
ERROR: 1411:11  comment-format  comment must start with a space
ERROR: 1412:16  expect          TypeScript@4.7 compile error:
This condition will always return 'false' since the types '"bar"' and '"baz"' have no overlap.
```


I'm realizing there's a less disruptive solution than splitting the test case so I'm going to change it